postgres-cmd-spec.json

```diff
@@ -0,0 +1,29 @@
+{
+  "label": "postgres command",
+  "environment": [
+    "PATH=/usr/lib64/ccache:/usr/bin:/usr/sbin:/bin:/sifive/tools/rust/stable/1.72.0/bin",
```

Comment:
This needs to be added somehow. I think a good way to go about this would be to make a new kind of wakebox runner so we can run arbitrary commands with whatever environment we pick.